Project-Euler/Project-Euler/Program.cs

```diff
@@ -20,19 +37,61 @@ public static void PrintIntro()
             //Add a couple more lines, maybe?
         }
 
-        public static void PrintMenuOptions() //This method will be private once I write the method that will call it. blah blah
+        public static int GetMenuSelection()
+        {
+            PrintMenuOptions();
+
+            while (true)
+            {
+                string userSelection = Console.ReadLine().ToUpper();
+
+                if (!string.IsNullOrEmpty(userSelection) && userSelection.Length == 1)
+                {
+                    char menuSelection = userSelection[0];
+
+                    if (menuSelection == 'A')
+                    {
+                        return 0;
+                    }
+                    else if (menuSelection == 'S')
+                    {
+                        return 1;
+                    }
+                    else if (menuSelection == 'Q')
+                    {
+                        return 2;
+                    }
+                }
+                else
+                {
```

Comment:
I would remove the _else_ because you want this code to execute even if it made it into the above if statement, but maybe they typed 'C' which wasn't valid.